### PR TITLE
Allow node 12.18

### PR DESCRIPTION
**Pull request by geoknee**
Adds node >12.18 to `engines` specification, and uses `12.18` on circle.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   },
   "engines": {
     "yarn": "^1.17.0",
-    "node": "~12.16.0"
+    "node": "^12.16.3 || >12.18"
```
Comment (lalexgap) [inline package.json:20]: Minor but could this just be `>=12.16.0`

Comment (geoknee) [inline package.json:20]:
```suggestion
    "node": ">=12.16.0"
```

File: packages/nitro-protocol/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
   },
   "engines": {
     "yarn": "^1.17.0",
-    "node": "~12.16.0"
+    "node": "^12.16.3 || >12.18"
```
Comment (geoknee) [inline packages/nitro-protocol/package.json:44]:
```suggestion
    "node": ">=12.16.0"
```

File: packages/wallet-core/package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "typescript": "3.7.5"
   },
   "engines": {
-    "node": "~12.16.0"
+    "node": "^12.16.3 || >12.18"
```
Comment (geoknee) [inline packages/wallet-core/package.json:23]:
```suggestion
    "node": ">=12.16.0"
```

File: packages/xstate-wallet/package.json [hunk 4]
```diff
@@ -117,7 +117,7 @@
     "wait-for-expect": "3.0.2"
   },
   "engines": {
-    "node": "~12.16.0"
+    "node": "^12.16.3 || >12.18"
```
Comment (geoknee) [inline packages/xstate-wallet/package.json:120]:
```suggestion
    "node": ">=12.16.0"
```

**Comment by andrewgordstewart** [comment]
Is there a benefit to using version 12.18?

**Comment by geoknee** [comment]
@andrewgordstewart yes, some projects we might want to integrate with could be using newer versions of node.

**Pull request merged**